deepstream/pravega-dev.Dockerfile

```diff
@@ -69,22 +69,18 @@ FROM builder-base as final
 # Copy over the cached dependencies
 COPY --from=cacher /usr/src/gstreamer-pravega/target target
 COPY --from=cacher /usr/local/cargo /usr/local/cargo
-COPY . .
-
+COPY Cargo.toml .
```

Comment:
We can optimize the order even further. Things that are last in the dependency chain should go last. For instance, if only integration-test changes, then it should only need to COPY integration-test and then build it. By copying everything first, you are preventing this type of optimization. You can improve this in this PR or a future one.

Reply:
Yes, that is my intent.